share/tools/web_config/webconfig.py

```diff
@@ -409,6 +409,8 @@ def parse_binding(self):
                             self.unget_char()
                             self.unget_char()
                             alt = True
+                    elif d=='\0':
```

Comment:
I've verified this works. In the future please pay attention to the surrounding style. I've added the missing whitespace around the `==` operator before merging it.